src/oph/heratepalvelu/tep/emailHandler.clj

```diff
@@ -11,18 +11,26 @@
             [oph.heratepalvelu.tep.tepCommon :as tc])
   (:import (software.amazon.awssdk.awscore.exception AwsServiceException)))
 
+;; Käsittelee TEP-jaksoja, joiden sähköpostit on määrä lähettää.
+
 (gen-class
   :name "oph.heratepalvelu.tep.emailHandler"
   :methods [[^:static handleSendTEPEmails
              [com.amazonaws.services.lambda.runtime.events.ScheduledEvent
               com.amazonaws.services.lambda.runtime.Context] void]])
 
-(defn bad-phone? [nippu]
+(defn bad-phone?
+  "Palauttaa true, jos nipun SMS-käsittelytilan mukaan puhelinnumero puuttuu tai
+  on virheellinen."
+  [nippu]
   (or (= (:phone-mismatch c/kasittelytilat) (:sms_kasittelytila nippu))
       (= (:no-phone c/kasittelytilat) (:sms_kasittelytila nippu))
       (= (:phone-invalid c/kasittelytilat) (:sms_kasittelytila nippu))))
 
-(defn lahetysosoite-update-item [nippu osoitteet]
+(defn lahetysosoite-update-item
+  "Päivittää nipun käsittelytilan tietokantaan, kun ei ole yksi kunnon
+  sähköpostiosoite. Parametri osoitteet on lista sähköpostiosoitteista."
```

Comment:
"Päivittää nipun käsittelytilan tietokantaan, kun ei ole yksi kunnon sähköpostiosoite." -> "Päivittää nipun käsittelytilan tietokantaan, kun osoitteista ei löydy selkeää vaihtoehtoa." 

Ehkä jotenkin näin.